### ISSUE TITLE
convert json validation output to the core json report format

### ISSUE THREAD

**Issue by gerrycampion** [cdisc-org/cdisc-rules-engine#1106]
Ensure we are able to generate excel reports from the json validation issues

**Comment by ASL-rmarshall** [comment]
The following components of the YAML rule definition should be be used/implemented when generating the output:

```yaml
Outcome
    Message
    Output Variables
Scope
```
